pallets/idn-manager/src/tests/xcm_controller.rs

```diff
@@ -0,0 +1,138 @@
+/*
+ * Copyright 2025 by Ideal Labs, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+//! Mock ExecuteController for fine-grained conditional error injection for xcm testing
+
+use frame_support::{dispatch::PostDispatchInfo, parameter_types};
+use xcm::{latest::prelude::*, v5::Assets, VersionedLocation, VersionedXcm, VersionedXcm::V5};
```

Comment:
`v5::Assets` is already in `xcm::latest::prelude::*`
```suggestion
use xcm::{latest::prelude::*, VersionedLocation, VersionedXcm, VersionedXcm::V5};
```